Plattform/gatsby-node.js

```diff
@@ -54,51 +54,56 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     const RootComponent = path.resolve(CATALOG_ROOT + 'CatalogRoot.tsx')
     const TopicComponent = path.resolve(CATALOG_ROOT + 'CatalogTopic.tsx')
     const LectureComponent = path.resolve(CATALOG_ROOT + 'CatalogLecture.tsx')
-    const TechnologyComponent = path.resolve(CATALOG_ROOT + 'CatalogTechnology.tsx')
```

Comment:
die Komponente "CatalogTechnology" brauchen wir für bspw. /UI-Entwicklung/React/
bitte entsprechend anpassen

Reply:
Ich dachte wir hatten gesagt wir nutzen diese Komponente nicht, da es einen Überblick über alle Lektionen unterhalb der Technologien in der Komponente CatalogTopic gibt?

Reply:
@fabianhinz please review